crates/transport-http/src/hyper_transport.rs

```diff
@@ -105,43 +122,63 @@ where
     }
 }
 
-impl<C, B> Service<RequestPacket> for &Http<Client<C, Full<B>>>
+impl<B, S> Http<HyperTransport<B, S>>
```

Comment:
hmm, why do we use `Http` here? it means we now have url both in `Http` and `HyperTransport`

Reply:
Resolved, `HyperTransport` now infers the url from `Http<T>`